### PR TITLE
[MLOB-1943] add vertex ai documentation under auto instrumentation doc

### PR DESCRIPTION
**Pull request by ncybul**
### What does this PR do? What is the motivation?

Adds Vertex AI to our list of auto-instrumented libraries for LLM Observability.

### Merge instructions

Merge readiness:
- [x] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/llm_observability/setup/auto_instrumentation.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Datadog's [LLM Observability Python SDK][16] provides integrations that automati
 | [Amazon Bedrock](#amazon-bedrock)          | >= 1.31.57         | >= 2.9.0          |
 | [Anthropic](#anthropic)                    | >= 0.28.0          | >= 2.10.0         |
 | [Google Gemini](#google-gemini)            | >= 0.7.2           | >= 2.14.0         |
+| [Vertex AI](#vertex-ai)                    | >= 1.71.1          | >= 2.18.0         |
```
Comment (ncybul) [inline content/en/llm_observability/setup/auto_instrumentation.md:31]: Need to verify the tracer release version.

**Comment by rtrieu** [comment]
Please let us know when the version has been verified and we will review/merge this PR!

File: content/en/llm_observability/setup/auto_instrumentation.md [hunk 2]
```diff
@@ -132,6 +133,22 @@ The Google Gemini integration instruments the following methods:
   - `model.generate_content()` (Also captures `chat.send_message()`)
   - `model.generate_content_async()` (Also captures `chat.send_message_async()`)
 
+## Vertex AI
+
+The Vertex AI integration provides automatic tracing for [Google's Vertex AI SDK for Python][21] content generation and chat message calls.
```
Comment (brett0000FF) [inline content/en/llm_observability/setup/auto_instrumentation.md:138]:
```suggestion
The Vertex AI integration automatically traces content generation and chat message calls made through [Google's Vertex AI Python SDK][21].
```

File: content/en/llm_observability/setup/auto_instrumentation.md (hunk 1 above)
Comment (brett0000FF) [reply content/en/llm_observability/setup/auto_instrumentation.md:31]: Were you able to confirm this version?

**Review by brett0000FF — COMMENTED** [review]
Just one small comment and suggestion! Let me know when you've addressed them so I can review/approve again. Thanks!

File: content/en/llm_observability/setup/auto_instrumentation.md (hunk 1 above)
Comment (ncybul) [reply content/en/llm_observability/setup/auto_instrumentation.md:31]: Yes, this is the correct version.

**Review by brett0000FF — APPROVED** [review]

**Pull request merged**